misc/start.sh

```diff
@@ -1,7 +1,24 @@
-#!/bin/bash
+#!/usr/bin/env bash
 # SPDX-License-Identifier: LGPL-3.0-or-later
 
-EXTRAOPTIONS=""
-[ -d "/ngrams" ] && EXTRAOPTIONS=" --languageModel /ngrams "
+EXTRAOPTIONS=${EXTRAOPTIONS:-}
+JAVAOPTIONS=${JAVAOPTIONS:-}
 
-java -cp languagetool-server.jar  org.languagetool.server.HTTPServer --port 8010 --public --allow-origin '*' ${EXTRAOPTIONS}
+if [ -d "/ngrams" ]; then
+    EXTRAOPTIONS="${EXTRAOPTIONS} --languageModel /ngrams"
+fi
+
+for var in ${!LT_*}; do
+  EXTRA_LT=true
+  echo "${var#'LT_'}="${!var} >> /tmp/config.properties
```

Comment:
Sorry for showing up so late to the party. 🙇
I have the problem here that I won't understand this "bashism" after half a year. I have been able to understand the code now, but it makes it harder for me and other developers and users to maintain or extend the code. Could you please rewrite that?

Could you please rewrite that?

Btw: I think the solution itself is okay, but I personally prefer a config file that I can put in. But your way and my way don't have to be mutually exclusive.

Reply:
@genofire FYI

Reply:
@silvio I tried to rewrite the bashism, but the result was even uglier to me. So I decided to add an explanatory comment instead.

Reply:
Well ... okay! :-D